aws-ecsfargate-terraform/variables.tf

```diff
@@ -1,15 +1,41 @@
 variable "aws_region" {
   type        = string
-  description = ""
-  default     = "us-east-1"
+  description = "The region of the AWS account where the SCIM bridge will be deployed."
 }
 
 variable "domain_name" {
   type        = string
-  description = ""
+  description = "The public DNS address pointing to your 1Password SCIM bridge."
 }
 
-variable "route53_zone_id" {
+variable "tags" {
+  type        = map(string)
+  description = "A set of tags to apply to all respective AWS resources."
+}
+
+variable "name_prefix" {
+  type        = string
+  description = "A default prefix to apply to the names of all AWS resources."
+}
+
+variable "vpc_name" {
   type        = string
-  description = ""
-}
\ No newline at end of file
+  description = "The name of an existing VPC to use."
+}
+
+variable "wildcard_cert" {
+  type        = bool
+  default     = false
+  description = "Set to true to use an existing wildcard certificate in ACM for the load balancer."
+}
+
+variable "using_route53" {
+  type        = bool
+  default     = true
+  description = "Set to false to use an external DNS provider"
+}
+
+variable "log_retention_days" {
+  type        = number
+  description = "Specifies the number of days to retain log events in CloudWatch. Set to the default of 0, the log is retained indefinitely."
```

Comment:
This variable is used to set the `retention_in_days` in the `aws_cloudwatch_log_group` resource block. If not specified for this resource, or if the value is set to 0, logs are kept indefinitely.

I've set it as 0 in the the `terraform.tfvars.template` file to reduce the amount of work needed to override the default, and make it intuitive. To override the default behaviour, change this value to something other than `0` (instead of having to add the variable manually or uncomment a line and add a value.

Reply:
Ah thanks for the clarification. Makes sense to me. :+1: